uaclient/entitlements/ros.py

```diff
@@ -3,11 +3,20 @@
 from uaclient.entitlements import repo
 
 
-class ROSESMEntitlement(repo.RepoEntitlement):
+class ROSCommonEntitlement(repo.RepoEntitlement):
     help_doc_url = "https://ubuntu.com/robotics/ros-esm"
-    name = "esm-ros"
-    title = "ROS ESM"
-    description = "ROS Extended Security Maintenance (ESM)"
-    repo_key_file = "ubuntu-advantage-esm-ros.gpg"
-    is_beta = True
+    repo_key_file = "ubuntu-advantage-ros-esm.gpg"
     _required_services = ("esm-infra", "esm-apps")  # type: Tuple[str, ...]
+    is_beta = True
+
+
+class ROSEntitlement(ROSCommonEntitlement):
+    name = "ros"
+    title = "ROS ESM Security Fixes"
+    description = "Robot Operating System security fixes"
+
+
+class ROSUpdatesEntitlement(ROSCommonEntitlement):
+    name = "ros-updates"
+    title = "ROS ESM All Updates"
+    description = "Robot Operating System security fixes and all updates"
```

Comment:
I tried to change the title and description of the services to concisely reflect what they each mean. Let me know if you have any suggestions to make them more clear :)

Reply:
@artivis and @GAN-Canonical what are your thoughts on these values?

The `title` field is used when referring to the service in a sentence. For example: "ROS ESM Security Fixes cannot be enabled with UA Apps: ESM disabled."
The `description` field shows up in the list of all services when you run `ua status`. For example:
```
SERVICE       ENTITLED  STATUS    DESCRIPTION
cc-eal        yes        disabled         Common Criteria EAL2 Provisioning Packages
cis           yes        disabled         Center for Internet Security Audit Tools
esm-apps      yes        disabled         UA Apps: Extended Security Maintenance (ESM)
esm-infra     yes        disabled         UA Infra: Extended Security Maintenance (ESM)
fips          yes        disabled         NIST-certified core packages
fips-updates  yes        disabled         NIST-certified core packages with priority security updates
livepatch     yes        disabled         Canonical Livepatch service
ros           yes        disabled         Robot Operating System security fixes
ros-updates   yes        disabled         Robot Operating System security fixes and all updates
```